### DISCUSSION TITLE
[BUG] Resource handler returned message: "Access denied for operation 'AWS::QuickSight::Dashboard'."

### DISCUSSION THREAD

**Discussion by vparmeland** [aws-solutions-library-samples/cloud-intelligence-dashboards-framework#158] [Q&A]
Hi,

CloudFormation ERROR : 

- MyStackName | ROLLBACK_IN_PROGRESS | The following resource(s) failed to create: [CUDOSDashboard, CIDDashboard]. Rollback requested by user.

- CUDOSDashboard | CREATE_FAILED | Resource handler returned message: "Access denied for operation 'AWS::QuickSight::Dashboard'." (RequestToken: defd7d3b-52d7-2570-7365-09b7e7d0b110, HandlerErrorCode: AccessDenied)
- CIDDashboard | CREATE_FAILED | Resource handler returned message: "Access denied for operation 'AWS::QuickSight::Dashboard'." (RequestToken: 87aea698-6321-dfa6-4ddd-2a946de55d8c, HandlerErrorCode: AccessDenied)

ERROR 
![image](https://user-images.githubusercontent.com/17825769/151513328-0b93e1c3-013d-4308-9acb-e3d64d9f044f.png)

Steps to reproduce : 

1. Follow the prerequisite steps then 

2. https://wellarchitectedlabs.com/cost/200_labs/200_cloud_intelligence/cost-usage-report-dashboards/dashboards/2b_cudos_dashboard/ 

3. CFN Deployment wizard
4. Failure during the CIDDashboard & CUDOSDashboard step

🙏

**Comment by vparmeland** [comment]
Any news ?

**Reply by darken99** [reply]
@cris-popa-aws can you take a look?

**Reply by vparmeland** [reply]
@cris-popa-aws @darken99 🆙 

Thanks

**Comment by aleem-99** [comment]
Any workaround available for this issue?

**Reply by darken99** [reply]
You have to check your permissions. If that doesn't help please open a Support Case to CloudFormation queue to find the reason.

**Reply by vparmeland** [reply]
@aleem-99 Same issue too ?

**Reply by vparmeland** [reply]
It's probably not a Cloudformation issue but a permission issue in the CUR CF Stack

**Reply by aleem-99** [reply]
I found the root cause, it was because of Quicksight being set to standard edition, after upgrading to Enterprise edition, stack is working now. CreateDashboard API on Quicksight is supported only for enterprise users.

**Reply by sd3z** [reply]
> I found the root cause, it was because of Quicksight being set to standard edition, after upgrading to Enterprise edition, stack is working now. CreateDashboard API on Quicksight is supported only for enterprise users.

This issue had me stuck for longer than I'd like to admit.   The fix of enabling Quicksights as Enterprise worked for me.  The documentation does mentioned under `Enable QuickSight:` Step 3 > After pressing the Sign up button you will be presented with 2 options, please ensure you select the **Enterprise Edition** during this step

**Comment by iakov-gan** [comment]
Close cleanup. Please comment if still an issue
